kvdb-rocksdb/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ The format is based on [Keep a Changelog].
   - `DatabaseConfig::default()` defaults to 1 column
   - `Database::with_columns` still accepts `u32`, but panics if `0` is provided 
   - `Database::open` panics if configuration with 0 columns is provided
+- Add `num_keys(col)` to get an estimate of the number of keys in a column (See [PR #285](https://github.com/paritytech/parity-common/pull/285)).
```

Comment:
idendetation

Reply:
I don't think this is wrong, the previous line is a sublist right?

Reply:
sorry my bad